installing/install_config/customizations.adoc

```diff
@@ -29,46 +29,55 @@ Custom Resource Definitions (CRDs) that are deployed on your cluster.
 |Configure the Kubernetes API server.
 
 |Authentication
-|You can configure the identity provider for your cluster.
+|You can configure the
+xref:../../authentication/understanding-identity-provider.adoc#understanding-identity-provider[identity provider]
+for your cluster.
 
 |Build
```

Comment:
@bmcelvee, do you have a link for this type?

Reply:
Finally merged!

xref:../../builds/build-configuration.adoc#build-configuration[build]